### PR TITLE
Tool controller

### PR DESCRIPTION
**Pull request by Belchy06**
- Adds Tool controller

### CODE REVIEW

File: projectunicorn-api/projectunicorn-api.csproj [hunk 1]
```diff
@@ -12,14 +12,11 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.1" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.0" />
```
Comment (roymoran) [inline projectunicorn-api/projectunicorn-api.csproj]: You won't need this package in this project, this is already included in the DataAccess project.

Comment (Belchy06) [reply projectunicorn-api/projectunicorn-api.csproj]: Fixed. I'll wait until all errors have been resolved before committing.

File: projectunicorn-api/Controllers/ToolsController.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            var tool = new Tool
+            {
+                Id = Tool.Id,
+                Name = Tool.Name,
+                Category = Tool.Category
+            };
+
+            _context.Tools.Add(tool);
+
+            await _context.SaveChangesAsync();
```
Comment (roymoran) [inline projectunicorn-api/Controllers/ToolsController.cs]: Your controller code looks good, my only comment is that we won't be doing data access here. Any code that accesses the database to create, read, update, or delete data will only happen in the DataAccess project. 

All functions here should look similar to `public async Task<ActionResult<ToolDto>> Get(Guid id)` where we call to the model to run additional logic and then we eventually call the data access project to persist any data.

Comment (roymoran) [reply projectunicorn-api/Controllers/ToolsController.cs]: I merged my member code even though it was work in progress just to show how I was getting data from the db through the model. That's mainly why I left my other controller functions empty.

**Pull request merged**